### ISSUE TITLE
Callout - FocusTrapZone - NVDA - Read content only 1 st time of focus

### ISSUE THREAD

**Issue by MdAmeerMind** [microsoft/fluentui#14021]
### Environment Information

- **Package version(s)**: Latest
- **Browser and OS versions**: Chrome + NVDA

### Describe the issue:
In callout we have use FocusTrapZone to trap focus, So content inside call out are read out by NVDA only when focus inner part first time. If we focus on link using tab it read aria label provided to link first time, again press tab focus goes to outer part again but narrator not read anything.

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/mdameer/pen/NWxLWrL?editors=0010

#### Actual behavior:

Narrator not reading content when focus is moving inside again and again

#### Expected behavior:

Content inside the callout should read all team if focus in moving inside again, not only first time

**Comment by xugao** [comment]
@MdAmeerMind - I cannot repro this issue with Narrator. I believe this is a bug with NVDA, not with the component itself.
I will mark the bug as external unless you can pin point what aria attributes we are setting incorrectly here

**Comment by MdAmeerMind** [comment]
Hi @xugao , thank you for reply. I have raise this issue with NVDA
